ts/src/bitmex.ts

```diff
@@ -1710,8 +1857,9 @@ export default class bitmex extends Exchange {
         const timestamp = this.parse8601 (this.safeString (order, 'timestamp'));
         const lastTradeTimestamp = this.parse8601 (this.safeString (order, 'transactTime'));
         const price = this.safeString (order, 'price');
-        const amount = this.safeString (order, 'orderQty');
-        const filled = this.safeString (order, 'cumQty');
+        const qty = this.safeString (order, 'orderQty');
+        const amount = this.convertFromRawQuantity (symbol, qty);
```

Comment:
```suggestion
        const amount = this.convertFromRawQuantity (symbol, this.safeString (order, 'orderQty'));
```
would be as it's below line. either this line needs to be changed or below line.

Reply:
@ttodua actually I don't like to inline too much because it makes harder to navigate with the debugger and it does not make any difference I think (both visual and practical)